src/status_im/navigation/handlers.cljs

```diff
@@ -50,6 +50,11 @@
   (fn [db [_ view-id]]
     (replace-view db view-id)))
 
+(defn- can-navigate-back? []
+  (and (not @(subscribe [:account-generation-in-progress?]))
```

Comment:
So we definitely have this information in the `db` parameter that is passed to handler, it doesn't look a good idea to subscribe inside the handler